vizro-core/src/vizro/static/css/loading.css

```diff
@@ -0,0 +1,40 @@
+/* Inspired by https://github.com/facultyai/dash-bootstrap-components/blob/5c8f4b40f1100fc00bf2d5c1d671a7815a6b2910/docs/static/loading.css */
+
+/* This creates a dark background in situations where neither dash-loading nor the Vizro app are displayed */
+html {
+  background: rgba(20, 23, 33, 1);
+  min-height: 100vh;
+}
+
+/* The dash-loading Div is present when Dash is initially loading, before the layout is built */
+
+/* The dash-loading-callback Div is present when Dash has loaded, but the layout is still building */
+
+/* Note that the dash-loading-callback Div is present until all elements are loaded, but as soon as the initial page
+elements (before on-page-load) are rendered, it gets pushed outside the viewable area, hence the spinner is not visible
+which is good, as we have individual loading spinners for elements.
+ */
+._dash-loading-callback,
+._dash-loading {
+  align-items: center;
```

Comment:
sorry for missing this. i think we can remove this

Reply:
No then the spinner is at the top and not the center anymore

Reply:
sorry! i was reading this wrong! yes you r right